finetuner/hubble.py

```diff
@@ -0,0 +1,62 @@
+from typing import List, Optional, Tuple, Union
+
+from docarray import DocumentArray
+
+from finetuner.client import FinetunerV1Client
+from finetuner.constants import EVAL_DATA, FINETUNED_MODELS_DIR, MODEL_IDS, TRAIN_DATA
+
+
+def push_data_to_hubble(
+    client: FinetunerV1Client,
+    train_data: Union[DocumentArray, str],
+    eval_data: Optional[Union[DocumentArray, str]],
+    experiment_name: str,
+    run_name: str,
+) -> Tuple[str, str]:
+    """Push DocumentArray for training and evaluation data on Hubble
+    if it's not already uploaded.
+    Note: for now, let's assume that we only receive `DocumentArray`-s.
+
+    :param client: The Finetuner API client.
+    :param train_data: Either a `DocumentArray` for training data that needs to be
+        pushed on Hubble or a name of the `DocumentArray` that is already uploaded.
+    :param eval_data: Either a `DocumentArray` for evaluation data that needs to be
+        pushed on Hubble or a name of the `DocumentArray` that is already uploaded.
+    :param experiment_name: Name of the experiment.
+    :param run_name: Name of the run.
+    :returns: Name(s) of pushed `DocumentArray`-s.
+    """
+    if isinstance(train_data, DocumentArray):
+        da_name = '-'.join(
```

Comment:
how about this function is only about a single document array and then you call it twice?